api/index.tsx

```diff
@@ -141,35 +141,35 @@ function Profile({
 }
 
 async function capture(fid?: number, buttonValue?: string, inputText?: string) {
-  if (fid) {
```

Comment:
@eliperelman this posthog capture section should probably be redone at some point because we only care about the fid of the interactor. The fid being viewed should be captured as a param of the event, but is not the user

Reply:
The fid being passed here is always the FID of the interactor.